src/index.ts

```diff
@@ -930,7 +930,7 @@ export class Bigtable {
  * that a callback is omitted.
  */
 promisifyAll(Bigtable, {
-  exclude: ['instance', 'operation', 'request'],
+  exclude: ['close', 'instance', 'operation', 'request'],
```

Comment:
In the absence of this change, the test just seems to hang forever.

Reply:
`promisifyAll` turns a `cb` based function into a promise based function, it looks like close was already a promise, and did not expect a `callback` as an argument:

```js
close(): Promise<void[]>
```

Reply:
Yes. I am making sure the reader of this PR knows why this change is necessary.